scripts/combine_metadata.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+from augur.utils import read_metadata
+from Bio import SeqIO
+import csv
+
+EMPTY = ''
+
+# This script was written in preparation for a future augur where commands
```

Comment:
Want to emphasise that as soon as `augur ...` can take multiple metadata inputs then this script can go away :)